fe1-web/parts/lao/organizer/eventCreation/events/CreateRollCall.tsx

```diff
@@ -25,37 +25,46 @@ function dateToTimestamp(date: Date): Timestamp {
 const CreateRollCall = ({ route }: any) => {
   const styles = route.params;
   const navigation = useNavigation();
-  const initialDate = new Date();
+  const initialStartDate = new Date();
```

Comment:
with this setup, it might be very easy for `proposed_start < creation` in the `requestCreateRollCall`.
You might want to set the `proposed_start` a few minutes into the future by default, or adjust it to be at least `now+5seconds` before you call `requestCreateRollCall`

Reply:
Good point! I set it to 5 minutes in the future by default now.